dev/stats/get_important_pr_candidates.py

```diff
@@ -267,9 +281,13 @@ def score(self):
         # If the body contains over 2000 characters, the PR should matter 40% more.
         # If the body contains fewer than 1000 characters, the PR should matter 20% less.
         #
+        if self.protm_score > 0:
+            interaction_score = self.interaction_score * 2
```

Comment:
I feel like having multiple #protm should weight more than having only one, maybe this can be linear with the number of #protm tags until a certain cap ? (just an idea)
```
interaction_score = self.interaction_score * min(self.protm_score + 1, 3)
```

If we do this, we should pay attention to not counting twice if the same person put two comments with #protm. (maintaining a set of people for instance)

Reply:
This change works for me. I will also add an explanation to the comments.